Clean build produces lots of warnings

12 on Windows:
```
D:\j\workspace\windows_nt_debug88592b53\Tools\packageresolve.targets(73,5): warning : Your project is not referencing the ".NETStandard,Version=v1.6" framework. Add a reference to ".NETStandard,Version=v1.6" in the "frameworks" section of your project.json, and then re-run NuGet restore. [D:\j\workspace\windows_nt_debug88592b53\src\System.AppContext\ref\System.AppContext.csproj]
D:\j\workspace\windows_nt_debug88592b53\Tools\packageresolve.targets(73,5): warning : Your project is not referencing the ".NETStandard,Version=v1.6" framework. Add a reference to ".NETStandard,Version=v1.6" in the "frameworks" section of your project.json, and then re-run NuGet restore. [D:\j\workspace\windows_nt_debug88592b53\src\System.AppContext\ref\System.AppContext.csproj]
D:\j\workspace\windows_nt_debug88592b53\Tools\packageresolve.targets(73,5): warning : Your project is not referencing the ".NETStandard,Version=v1.6" framework. Add a reference to ".NETStandard,Version=v1.6" in the "frameworks" section of your project.json, and then re-run NuGet restore. [D:\j\workspace\windows_nt_debug88592b53\src\System.AppContext\ref\System.AppContext.csproj]
D:\j\workspace\windows_nt_debug88592b53\Tools\packageresolve.targets(73,5): warning : Your project is not referencing the ".NETStandard,Version=v1.6" framework. Add a reference to ".NETStandard,Version=v1.6" in the "frameworks" section of your project.json, and then re-run NuGet restore. [D:\j\workspace\windows_nt_debug88592b53\src\System.AppContext\ref\System.AppContext.csproj]
D:\j\workspace\windows_nt_debug88592b53\Tools\packageresolve.targets(73,5): warning : Your project is not referencing the ".NETStandard,Version=v1.6" framework. Add a reference to ".NETStandard,Version=v1.6" in the "frameworks" section of your project.json, and then re-run NuGet restore. [D:\j\workspace\windows_nt_debug88592b53\src\System.AppContext\src\System.AppContext.csproj]
D:\j\workspace\windows_nt_debug88592b53\Tools\packageresolve.targets(73,5): warning : Your project is not referencing the ".NETStandard,Version=v1.6" framework. Add a reference to ".NETStandard,Version=v1.6" in the "frameworks" section of your project.json, and then re-run NuGet restore. [D:\j\workspace\windows_nt_debug88592b53\src\System.AppContext\src\System.AppContext.csproj]
D:\j\workspace\windows_nt_debug88592b53\Tools\packageresolve.targets(73,5): warning : Your project is not referencing the ".NETStandard,Version=v1.6" framework. Add a reference to ".NETStandard,Version=v1.6" in the "frameworks" section of your project.json, and then re-run NuGet restore. [D:\j\workspace\windows_nt_debug88592b53\src\System.AppContext\src\System.AppContext.csproj]
D:\j\workspace\windows_nt_debug88592b53\Tools\packageresolve.targets(73,5): warning : Your project is not referencing the ".NETStandard,Version=v1.6" framework. Add a reference to ".NETStandard,Version=v1.6" in the "frameworks" section of your project.json, and then re-run NuGet restore. [D:\j\workspace\windows_nt_debug88592b53\src\System.AppContext\src\System.AppContext.csproj]
C:\Program Files (x86)\MSBuild\14.0\bin\Microsoft.Common.CurrentVersion.targets(1819,5): warning MSB3277: Found conflicts between different versions of the same dependent assembly that could not be resolved.  These reference conflicts are listed in the build log when log verbosity is set to detailed. [D:\j\workspace\windows_nt_debug88592b53\src\System.Net.Http\src\System.Net.Http.csproj]
C:\Program Files (x86)\MSBuild\14.0\bin\Microsoft.Common.CurrentVersion.targets(1819,5): warning MSB3277: Found conflicts between different versions of the same dependent assembly that could not be resolved.  These reference conflicts are listed in the build log when log verbosity is set to detailed. [D:\j\workspace\windows_nt_debug88592b53\src\System.Security.Cryptography.OpenSsl\ref\System.Security.Cryptography.OpenSsl.csproj]
C:\Program Files (x86)\MSBuild\14.0\bin\Microsoft.Common.CurrentVersion.targets(1819,5): warning MSB3277: Found conflicts between different versions of the same dependent assembly that could not be resolved.  These reference conflicts are listed in the build log when log verbosity is set to detailed. [D:\j\workspace\windows_nt_debug88592b53\src\System.Security.Cryptography.X509Certificates\src\System.Security.Cryptography.X509Certificates.csproj]
C:\Program Files (x86)\MSBuild\14.0\bin\Microsoft.Common.CurrentVersion.targets(1819,5): warning MSB3277: Found conflicts between different versions of the same dependent assembly that could not be resolved.  These reference conflicts are listed in the build log when log verbosity is set to detailed. [D:\j\workspace\windows_nt_debug88592b53\src\System.Security.Cryptography.X509Certificates\src\System.Security.Cryptography.X509Certificates.csproj]

    12 Warning(s)
    0 Error(s)
```

27 on Ubuntu:
```
/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/Tools/packageresolve.targets(73,5): warning : Your project is not referencing the ".NETStandard,Version=v1.6" framework. Add a reference to ".NETStandard,Version=v1.6" in the "frameworks" section of your project.json, and then re-run NuGet restore. [/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.AppContext/ref/System.AppContext.csproj]
/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/Tools/packageresolve.targets(73,5): warning : Your project is not referencing the ".NETStandard,Version=v1.6" framework. Add a reference to ".NETStandard,Version=v1.6" in the "frameworks" section of your project.json, and then re-run NuGet restore. [/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.AppContext/ref/System.AppContext.csproj]
/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/Tools/packageresolve.targets(73,5): warning : Your project is not referencing the ".NETStandard,Version=v1.6" framework. Add a reference to ".NETStandard,Version=v1.6" in the "frameworks" section of your project.json, and then re-run NuGet restore. [/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.AppContext/ref/System.AppContext.csproj]
/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/Tools/packageresolve.targets(73,5): warning : Your project is not referencing the ".NETStandard,Version=v1.6" framework. Add a reference to ".NETStandard,Version=v1.6" in the "frameworks" section of your project.json, and then re-run NuGet restore. [/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.AppContext/ref/System.AppContext.csproj]
/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/Tools/packageresolve.targets(73,5): warning : Your project is not referencing the ".NETStandard,Version=v1.6" framework. Add a reference to ".NETStandard,Version=v1.6" in the "frameworks" section of your project.json, and then re-run NuGet restore. [/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.AppContext/src/System.AppContext.csproj]
/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/Tools/packageresolve.targets(73,5): warning : Your project is not referencing the ".NETStandard,Version=v1.6" framework. Add a reference to ".NETStandard,Version=v1.6" in the "frameworks" section of your project.json, and then re-run NuGet restore. [/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.AppContext/src/System.AppContext.csproj]
/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/Tools/packageresolve.targets(73,5): warning : Your project is not referencing the ".NETStandard,Version=v1.6" framework. Add a reference to ".NETStandard,Version=v1.6" in the "frameworks" section of your project.json, and then re-run NuGet restore. [/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.AppContext/src/System.AppContext.csproj]
/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/Tools/packageresolve.targets(73,5): warning : Your project is not referencing the ".NETStandard,Version=v1.6" framework. Add a reference to ".NETStandard,Version=v1.6" in the "frameworks" section of your project.json, and then re-run NuGet restore. [/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.AppContext/src/System.AppContext.csproj]
/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.ComponentModel.TypeConverter/src/Resources/Strings.resx : warning MSB3568: Duplicate resource name "TypeDescriptorProviderError" is not allowed, ignored. [/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.ComponentModel.TypeConverter/src/System.ComponentModel.TypeConverter.csproj]
/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.ComponentModel.TypeConverter/src/Resources/Strings.resx : warning MSB3568: Duplicate resource name "TypeDescriptorUnsupportedRemoteObject" is not allowed, ignored. [/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.ComponentModel.TypeConverter/src/System.ComponentModel.TypeConverter.csproj]
/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.ComponentModel.TypeConverter/src/Resources/Strings.resx : warning MSB3568: Duplicate resource name "TypeDescriptorProviderError" is not allowed, ignored. [/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.ComponentModel.TypeConverter/src/System.ComponentModel.TypeConverter.csproj]
/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.ComponentModel.TypeConverter/src/Resources/Strings.resx : warning MSB3568: Duplicate resource name "TypeDescriptorUnsupportedRemoteObject" is not allowed, ignored. [/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.ComponentModel.TypeConverter/src/System.ComponentModel.TypeConverter.csproj]
/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.ComponentModel.TypeConverter/src/Resources/Strings.resx : warning MSB3568: Duplicate resource name "TypeDescriptorProviderError" is not allowed, ignored. [/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.ComponentModel.TypeConverter/src/System.ComponentModel.TypeConverter.csproj]
/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.ComponentModel.TypeConverter/src/Resources/Strings.resx : warning MSB3568: Duplicate resource name "TypeDescriptorUnsupportedRemoteObject" is not allowed, ignored. [/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.ComponentModel.TypeConverter/src/System.ComponentModel.TypeConverter.csproj]
/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.ComponentModel.TypeConverter/src/Resources/Strings.resx : warning MSB3568: Duplicate resource name "TypeDescriptorProviderError" is not allowed, ignored. [/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.ComponentModel.TypeConverter/src/System.ComponentModel.TypeConverter.csproj]
/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.ComponentModel.TypeConverter/src/Resources/Strings.resx : warning MSB3568: Duplicate resource name "TypeDescriptorUnsupportedRemoteObject" is not allowed, ignored. [/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.ComponentModel.TypeConverter/src/System.ComponentModel.TypeConverter.csproj]
/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/Tools/Microsoft.Common.CurrentVersion.targets(1819,5): warning MSB3277: Found conflicts between different versions of the same dependent assembly that could not be resolved.  These reference conflicts are listed in the build log when log verbosity is set to detailed. [/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.Net.Http/src/System.Net.Http.csproj]
/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/Tools/Microsoft.Common.CurrentVersion.targets(1819,5): warning MSB3277: Found conflicts between different versions of the same dependent assembly that could not be resolved.  These reference conflicts are listed in the build log when log verbosity is set to detailed. [/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.Security.Cryptography.OpenSsl/ref/System.Security.Cryptography.OpenSsl.csproj]
/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/Tools/Microsoft.Common.CurrentVersion.targets(1819,5): warning MSB3277: Found conflicts between different versions of the same dependent assembly that could not be resolved.  These reference conflicts are listed in the build log when log verbosity is set to detailed. [/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.Security.Cryptography.X509Certificates/src/System.Security.Cryptography.X509Certificates.csproj]
/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/Tools/Microsoft.Common.CurrentVersion.targets(1819,5): warning MSB3277: Found conflicts between different versions of the same dependent assembly that could not be resolved.  These reference conflicts are listed in the build log when log verbosity is set to detailed. [/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.Security.Cryptography.X509Certificates/src/System.Security.Cryptography.X509Certificates.csproj]
libust[33786/33786] : warning : HOME environment variable not set. Disabling LTTng-UST per-user tracing. (in setup_local_apps() at lttng-ust-comm.c:305) [/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.Globalization/tests/System.Globalization.Tests.csproj]
libust[33796/33796] : warning : HOME environment variable not set. Disabling LTTng-UST per-user tracing. (in setup_local_apps() at lttng-ust-comm.c:305) [/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.Globalization/tests/System.Globalization.Tests.csproj]
libust[33823/33823] : warning : HOME environment variable not set. Disabling LTTng-UST per-user tracing. (in setup_local_apps() at lttng-ust-comm.c:305) [/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.Globalization/tests/System.Globalization.Tests.csproj]
libust[33833/33833] : warning : HOME environment variable not set. Disabling LTTng-UST per-user tracing. (in setup_local_apps() at lttng-ust-comm.c:305) [/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.Globalization/tests/System.Globalization.Tests.csproj]
libust[33839/33839] : warning : HOME environment variable not set. Disabling LTTng-UST per-user tracing. (in setup_local_apps() at lttng-ust-comm.c:305) [/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.Globalization/tests/System.Globalization.Tests.csproj]
libust[33862/33862] : warning : HOME environment variable not set. Disabling LTTng-UST per-user tracing. (in setup_local_apps() at lttng-ust-comm.c:305) [/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.Globalization/tests/System.Globalization.Tests.csproj]
libust[33869/33869] : warning : HOME environment variable not set. Disabling LTTng-UST per-user tracing. (in setup_local_apps() at lttng-ust-comm.c:305) [/mnt/j/workspace/dotnet_corefx/master/ubuntu14.04_debug/src/System.Globalization/tests/System.Globalization.Tests.csproj]

    27 Warning(s)
    0 Error(s)
```

After my fixes, Matt's fixes, and Alex's fixes, it looks like we're down to just these four warnings remaining:
```
C:\Program Files (x86)\MSBuild\14.0\bin\Microsoft.Common.CurrentVersion.targets(1819,5): warning MSB3277: Found conflicts between different versions of the same dependent assembly that could not be resolved.  These reference conflicts are listed in the build log when log verbosity is set to detailed. [D:\j\workspace\windows_nt_debug88592b53\src\System.Net.Http\src\System.Net.Http.csproj]
C:\Program Files (x86)\MSBuild\14.0\bin\Microsoft.Common.CurrentVersion.targets(1819,5): warning MSB3277: Found conflicts between different versions of the same dependent assembly that could not be resolved.  These reference conflicts are listed in the build log when log verbosity is set to detailed. [D:\j\workspace\windows_nt_debug88592b53\src\System.Security.Cryptography.OpenSsl\ref\System.Security.Cryptography.OpenSsl.csproj]
C:\Program Files (x86)\MSBuild\14.0\bin\Microsoft.Common.CurrentVersion.targets(1819,5): warning MSB3277: Found conflicts between different versions of the same dependent assembly that could not be resolved.  These reference conflicts are listed in the build log when log verbosity is set to detailed. [D:\j\workspace\windows_nt_debug88592b53\src\System.Security.Cryptography.X509Certificates\src\System.Security.Cryptography.X509Certificates.csproj]
C:\Program Files (x86)\MSBuild\14.0\bin\Microsoft.Common.CurrentVersion.targets(1819,5): warning MSB3277: Found conflicts between different versions of the same dependent assembly that could not be resolved.  These reference conflicts are listed in the build log when log verbosity is set to detailed. [D:\j\workspace\windows_nt_debug88592b53\src\System.Security.Cryptography.X509Certificates\src\System.Security.Cryptography.X509Certificates.csproj]
```
@bartonjs, since these are mostly crypto, could you take a look?

--- a/src/System.Net.Http/src/unix/project.json
+++ b/src/System.Net.Http/src/unix/project.json
@@ -16,7 +16,7 @@
         "System.Diagnostics.Tools": "4.0.0",
         "System.Globalization": "4.0.10",
         "System.Globalization.Extensions": "4.0.0",
-        "System.IO": "4.0.10",
+        "System.IO": "4.1.0-rc4-24207-03",
         "System.IO.FileSystem": "4.0.0",
         "System.IO.Compression": "4.0.0",
         "System.Net.Primitives": "4.0.10",

--- a/src/System.Security.Cryptography.OpenSsl/ref/project.json
+++ b/src/System.Security.Cryptography.OpenSsl/ref/project.json
@@ -1,8 +1,8 @@
 {
   "dependencies": {
-    "System.Runtime": "4.0.0",
+    "System.Runtime": "4.1.0-rc4-24207-03",
     "System.Runtime.Handles": "4.0.0",
-    "System.IO": "4.0.0",
+    "System.IO": "4.1.0-rc4-24207-03",
     "System.Security.Cryptography.Algorithms": "4.2.0-rc4-24207-03",
     "System.Security.Cryptography.Primitives": "4.0.0-rc4-24207-03"
   },

--- a/src/System.Security.Cryptography.X509Certificates/src/unix/project.json
+++ b/src/System.Security.Cryptography.X509Certificates/src/unix/project.json
@@ -8,7 +8,7 @@
     "System.Globalization.Calendars": "4.0.0",
     "System.IO.FileSystem": "4.0.0",
     "System.Resources.ResourceManager": "4.0.0",
-    "System.Runtime": "4.0.20",
+    "System.Runtime": "4.1.0-rc4-24207-03",
     "System.Runtime.Handles": "4.0.0",
     "System.Runtime.InteropServices": "4.0.20",
     "System.Runtime.Numerics": "4.0.0",

--- a/src/System.Security.Cryptography.X509Certificates/src/win/project.json
+++ b/src/System.Security.Cryptography.X509Certificates/src/win/project.json
@@ -9,7 +9,7 @@
     "System.IO.FileSystem": "4.0.0",
     "System.IO.FileSystem.Watcher": "4.0.0-rc4-24207-03",
     "System.Resources.ResourceManager": "4.0.0",
-    "System.Runtime": "4.0.20",
+    "System.Runtime": "4.1.0-rc4-24207-03",
     "System.Runtime.Handles": "4.0.0",
     "System.Runtime.InteropServices": "4.0.20",
     "System.Runtime.Numerics": "4.0.0",

